### ISSUE TITLE
Decode kernel kernel oopses/warns

### ISSUE THREAD

**Issue by broonie** [kernelci/kernelci-core#1637]
In order to improve the immediate usability of kernel logs with backtraces in it is helpful to demangle the addresses printed to show actual code lines. There is a tool in the kernel to do this which is run like so:

  CROSS_COMPILE=aarch64-none-linux-gnu- ./scripts/decode_stacktrace.sh vlinux < raw.log > parsed.log

(note that it needs vmlinux which we don't currently save, and a toolchain). The simplest thing would just be to run all logs through the script after they've been posted, it'd be a bit expensive o fire up containers for it though so e might want to be smarter about it.

**Comment by broonie** [comment]
For clang builds CROSS_COMPILE needs to be set to invoke the LLVM tools - CROSS_COMPILE=llvm- *should* do it.

**Comment by broonie** [comment]
https://github.com/kernelci/kernelci-core/pull/1639

**Comment by gctucker** [comment]
That sounds like a feature we could add once the new API & Pipeline are getting ready for production.

**Comment by gctucker** [comment]
In fact we could have a service that will automatically rebuild the kernel with extra debug options and run tests and parse the stacktrace etc. when a kernel problem has been detected.

**Comment by broonie** [comment]
Yes, I filed this as an issue rather than looking at implementing since it felt like a pipeline thing.

Trying with debug configs might be useful too, though there's always the risk that the debug options peturb the test of course.
